.changeset/yummy-owls-cough.md

```diff
@@ -0,0 +1,13 @@
+---
+'@commercetools/composable-commerce-test-data': minor
+---
+
+We're introducing a new model named `DeliveryItemGraphql` that can be consumed from the `@commercetools/composable-commerce-test-data/order` entry point.
```

Comment:
I think this can be misleading as we're not introducing a new model but migrating an existing one which still exports a GraphQL, REST and compat versions.